### ISSUE TITLE
AREADEF & ROOMDEF.Clients

### ISSUE THREAD

**Issue by xMirroRx** [Sphereserver/Source-X#1064]
REGION.CLIENTS "It should get the total number of users in the region."
ROOM.CLIENTS "It should only get the number of users in the room".

**Comment by canerksk** [comment]
This is already available.

https://wiki.spherecommunity.net/index.php?title=Regions
![image](https://user-images.githubusercontent.com/10347131/233783777-e0c37984-8caf-4d2d-9803-9b3a30342f80.png)


https://wiki.spherecommunity.net/index.php?title=Rooms
![image](https://user-images.githubusercontent.com/10347131/233783766-b8bec963-8a34-459c-8dd7-6b751957ed27.png)

**Comment by xMirroRx** [comment]
Yes, I know all of them. the problem is exactly that room.clients not only gives the number of players in the room but also covers the players outside the room i.e. the whole region same as region.clients

**Comment by xwerswoodx** [comment]
Yep I confirmed the bug, the issue is ROOM.CLIENTS and REGION.CLIENTS returns the CLIENTS count for sector, not the room and region itself. Probably broken after sector update.
